assembl/models/auth.py

```diff
@@ -630,6 +632,62 @@ class AgentStatusInDiscussion(DiscussionBoundBase):
     first_subscribed = Column(DateTime)
     last_unsubscribed = Column(DateTime)
     user_created_on_this_discussion = Column(Boolean, server_default='0')
+    accepted_cookies = Column(Text())  # JSON blob
+
+    def __init__(self, *args, **kwargs):
+        super(AgentStatusInDiscussion, self).__init__(*args, **kwargs)
+        self._convert_cookies_to_enums()
+
+    @orm.reconstructor
+    def init_on_load(self):
+        self._convert_cookies_to_enums()
+
+    def _convert_cookies_to_enums(self):
+        # A private variable _accepted_cookies is used to track the Enum-list of cookies supported
+        if '_accepted_cookies' not in vars(self):
+            if not self.accepted_cookies:
+                self._accepted_cookies = list()
+            else:
+                self._accepted_cookies = map(
```

Comment:
see my comment above about list comprehensions for the 2 `map()` here (it's a matter of preference but when I read python code, I expect list comprehensions)